source/developer/dev-setup-centos-7.rst

```diff
@@ -0,0 +1,98 @@
+.. _dev-setup-centos-7:
+
+Installing Developer Components on CentOS 7
+===========================================
+
+Set up your development environment for building, running, and testing Mattermost.
+
+1. If you are developing with the Docker container, install and configure Docker CE:
+
+  a. Follow the Docker installation instructions at https://docs.docker.com/engine/installation/linux/docker-ce/centos/.
+
+  b. Edit your ``/etc/hosts`` file to include the following line:
+
+    ``127.0.0.1     dockerhost``
+
+  c. Add your username to the *docker* group so that you can run Docker without using sudo. Replace *{username}* with your username.
+
+    ``sudo gpasswd -a {username} docker``
+
+  d. Restart the Docker daemon.
+
+    ``sudo service docker restart``
```

Comment:
@JeffSchering both ways work, the system will automatically convert from system to systermctl